src/CMakeLists.txt

```diff
@@ -340,14 +280,6 @@ target_link_libraries(scinsynth
     spdlog
 )
 
-target_include_directories(scinsynth PRIVATE
-    ${AVCODEC_INCLUDE_DIR}
-    ${AVFORMAT_INCLUDE_DIR}
-    ${AVUTIL_INCLUDE_DIR}
-    ${Vulkan_INCLUDE_DIRS}
-    ${SWSCALE_INCLUDE_DIR}
```

Comment:
i don't think these were doing anything before -- find_library just finds a library file; but it's find_package with an appropriately written CMake find-module that will often set "lib_INCLUDE_DIR". from what i can tell, things just worked before because everything was unpacked into the build dir.

Reply:
Could be the case, always better to remove cruft like this if we can. Probably those are there from historical times when Scintillator included those dependencies in third_party as submodules.